<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageNotes</class>
 <widget class="QDialog" name="ManageNotes">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Gestion notes</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="add_button">
       <property name="text">
        <string notr="true">Ajouter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="del_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string notr="true">Supprimer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="ManageNotesList" name="note_list">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QPushButton" name="group_action_button">
     <property name="text">
      <string notr="true">Faire des actions sur plusieurs notes</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="0">
      <widget class="QLabel" name="name_label">
       <property name="text">
        <string notr="true">&amp;Nom:</string>
       </property>
       <property name="buddy">
        <cstring>name_input</cstring>
       </property>
      </widget>
     </item>
     <item row="25" column="2">
      <widget class="QLabel" name="notes_infos">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="28" column="0" colspan="3">
      <widget class="QListWidget" name="category_list"/>
     </item>
     <item row="33" column="1" colspan="2">
      <widget class="QPushButton" name="save_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string notr="true">Sauvegarder</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="nickname_label">
       <property name="text">
        <string notr="true">S&amp;urnom: </string>
       </property>
       <property name="buddy">
        <cstring>nickname_input</cstring>
       </property>
      </widget>
     </item>
     <item row="15" column="0" colspan="3">
      <widget class="QLabel" name="promo_label">
       <property name="text">
        <string notr="true">Promo:</string>
       </property>
       <property name="buddy">
        <cstring>promo_input</cstring>
       </property>
      </widget>
     </item>
     <item row="17" column="0" colspan="2">
      <widget class="QLabel" name="mail_label">
       <property name="text">
        <string notr="true">Mai&amp;l:</string>
       </property>
       <property name="buddy">
        <cstring>mail_input</cstring>
       </property>
      </widget>
     </item>
     <item row="22" column="0" colspan="3">
      <widget class="QLabel" name="photo_label">
       <property name="text">
        <string notr="true">Photo:</string>
       </property>
       <property name="buddy">
        <cstring>photo_button</cstring>
       </property>
      </widget>
     </item>
     <item row="33" column="0">
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string notr="true">Annuler</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0" colspan="3">
      <widget class="Input" name="birthdate_input">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="19" column="0" colspan="3">
      <widget class="Input" name="mail_input">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="29" column="0">
      <widget class="QComboBox" name="category_selector">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAlphabetically</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="3">
      <widget class="Input" name="name_input">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="23" column="0" colspan="3">
      <widget class="QPushButton" name="photo_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string notr="true">Modifier la photo</string>
       </property>
      </widget>
     </item>
     <item row="27" column="0" colspan="3">
      <widget class="QPushButton" name="cotiz_button">
       <property name="text">
        <string notr="true">Cotiz</string>
       </property>
      </widget>
     </item>
     <item row="26" column="0" colspan="3">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="29" column="1">
      <widget class="QPushButton" name="add_category_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string notr="true">Ajouter à la catégorie</string>
       </property>
      </widget>
     </item>
     <item row="20" column="0" colspan="2">
      <widget class="QLabel" name="phone_label">
       <property name="text">
        <string notr="true">&amp;Téléphone:</string>
       </property>
       <property name="buddy">
        <cstring>phone_input</cstring>
       </property>
      </widget>
     </item>
     <item row="25" column="0">
      <widget class="QLabel" name="photo">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>160</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="Input" name="nickname_input">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="29" column="2">
      <widget class="QPushButton" name="remove_category_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string notr="true">Enlever de la catégorie</string>
       </property>
      </widget>
     </item>
     <item row="21" column="0" colspan="3">
      <widget class="Input" name="phone_input">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="16" column="0" colspan="3">
      <widget class="QComboBox" name="promo_input">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string notr="true">1A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">2A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">3A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">3S</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">4A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">5A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Externe</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">ESIAB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Prof</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Ancien</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="Input" name="first_name_input">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QLabel" name="birthdate_label">
       <property name="text">
        <string notr="true">&amp;Date de naissance: </string>
       </property>
       <property name="buddy">
        <cstring>birthdate_input</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="first_name_label">
       <property name="text">
        <string notr="true">Pr&amp;énom: </string>
       </property>
       <property name="buddy">
        <cstring>first_name_input</cstring>
       </property>
      </widget>
     </item>
     <item row="24" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="mails_checkbox">
         <property name="text">
          <string notr="true">Mails</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="stats_checkbox">
         <property name="text">
          <string notr="true">Stats</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="agios_checkbox">
         <property name="text">
          <string notr="true">Agios</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Input</class>
   <extends>QLineEdit</extends>
   <header>gui/input_widget</header>
  </customwidget>
  <customwidget>
   <class>ManageNotesList</class>
   <extends>QListWidget</extends>
   <header>gui/notes_management_window</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nickname_input</tabstop>
  <tabstop>first_name_input</tabstop>
  <tabstop>name_input</tabstop>
  <tabstop>birthdate_input</tabstop>
  <tabstop>promo_input</tabstop>
  <tabstop>mail_input</tabstop>
  <tabstop>phone_input</tabstop>
  <tabstop>photo_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>photo_button</sender>
   <signal>clicked()</signal>
   <receiver>ManageNotes</receiver>
   <slot>add_photo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_button</sender>
   <signal>clicked()</signal>
   <receiver>ManageNotes</receiver>
   <slot>add_fnc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>ManageNotes</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_button</sender>
   <signal>clicked()</signal>
   <receiver>ManageNotes</receiver>
   <slot>save_fnc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>note_list</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>ManageNotes</receiver>
   <slot>on_note_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>del_button</sender>
   <signal>clicked()</signal>
   <receiver>ManageNotes</receiver>
   <slot>del_fnc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_category_button</sender>
   <signal>pressed()</signal>
   <receiver>ManageNotes</receiver>
   <slot>add_category_fnc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>629</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remove_category_button</sender>
   <signal>pressed()</signal>
   <receiver>ManageNotes</receiver>
   <slot>remove_category_fnc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>629</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cotiz_button</sender>
   <signal>clicked()</signal>
   <receiver>ManageNotes</receiver>
   <slot>cotiz_fnc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>group_action_button</sender>
   <signal>clicked()</signal>
   <receiver>ManageNotes</receiver>
   <slot>group_action_fnc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>786</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add_photo()</slot>
  <slot>add_fnc()</slot>
  <slot>save_fnc()</slot>
  <slot>on_note_selected()</slot>
  <slot>del_fnc()</slot>
  <slot>add_category_fnc()</slot>
  <slot>remove_category_fnc()</slot>
  <slot>cotiz_fnc()</slot>
  <slot>group_action_fnc()</slot>
  <slot>trigger_search()</slot>
 </slots>
</ui>
